pkg/lib/aws/ec2.go

```diff
@@ -329,3 +330,54 @@ func (c *Client) DescribeVpcs() ([]ec2.Vpc, error) {
 
 	return vpcs, nil
 }
+
+func (c *Client) ListVolumes() ([]ec2.Volume, error) {
+	var volumes []ec2.Volume
+	err := c.EC2().DescribeVolumesPages(&ec2.DescribeVolumesInput{}, func(output *ec2.DescribeVolumesOutput, lastPage bool) bool {
+		if output == nil {
+			return false
+		}
+		for _, volume := range output.Volumes {
+			if volume == nil {
+				continue
+			}
+			volumes = append(volumes, *volume)
+		}
+
+		return true
+	})
+
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	return volumes, nil
+}
+
+func (c *Client) ListPVCVolumesForCluster(clusterName string) ([]ec2.Volume, error) {
+	var volumes []ec2.Volume
+	err := c.EC2().DescribeVolumesPages(&ec2.DescribeVolumesInput{}, func(output *ec2.DescribeVolumesOutput, lastPage bool) bool {
+		if output == nil {
+			return false
+		}
+		for _, volume := range output.Volumes {
+			if volume == nil {
+				continue
+			}
```

Comment:
these two can be merged into a single function, or at least `ListPVCVolumesForCluster` can call `ListVolumes`

The single signature would look like:

```go
func (c *Client) ListVolumes(tags... *ec2.Tag) ([]ec2.Volume, error) {
    // (...)
}
```